extensions/amp-live-list/amp-live-list.md

```diff
@@ -101,9 +101,9 @@ polling interval.
 
 ### update
 
-The `update` reference point is hidden by default (through an `.amp-hidden`
-class which can be overriden) and is shown when new items are received from
-the server. You may style this reference point as a `fixed` position
+The `update` reference point is shown when new items are received from
+the server, to provide an affordance for users to refresh the page with new items when they are available. It is hidden by default (through an `.amp-hidden`
+class which can be overriden). You may style this reference point as a `fixed` position
```

Comment:
can we expand this to "through an `.amp-hidden` class whose style can be overridden"